contracts/src/token/erc721/mod.rs

```diff
@@ -150,6 +153,8 @@ pub enum Error {
     InvalidOperator(ERC721InvalidOperator),
 }
 
+impl_method_error!(Error);
```

Comment:
I prefer we write it here explicitly, since this should be something that the SDK exposes as a `derive` or by default. It makes it easier to change later.